SUPPORTED_PLATFORMS.md

```diff
@@ -0,0 +1,46 @@
+# Supported platforms
+
+|  System | Support type | Supported versions | Notes |
+|---|---|---|---|
+| Linux | Tier 1 | Linux >= 2.6.32 with glibc >= 2.12 | |
+| macOS | Tier 1 | macOS >= 10.15 | Current and previous macOS release |
+| Windows | Tier 1 | >= Windows 8 | VS 2015 and later are supported |
+| FreeBSD | Tier 1 | >= 10 | |
```

Comment:
also openBSD

Reply:
Not sure what the minimum version for openbsd is since libuv hasn't listed it either. An empty value or an N/A is good enough for now perhaps?

Reply:
Tier 3: Not tested and no guarantees, but anecdotally reported to work.

(Eventually; right now it's Tier 1, since CI tests it. LuaJIT is _not_ a required dependency; Neovim builds perfectly fine with PUC Lua.)

Reply:
No no, I meant the minimum support version of openbsd, not its tier. Anyway, I'll figure out an appropriate solution :+1: